### PR TITLE
Changes needed during compute info runtime

### PR DESCRIPTION
**Pull request by jkgoodrich**
Also fixes issue of adj annotation for trio stats and changes annotation HT to allow for easy comparison of info methods

### CODE REVIEW

File: gnomad_qc/v4/annotations/generate_variant_qc_annotations.py [hunk 1]
```diff
@@ -381,14 +479,37 @@ def create_variant_qc_annotation_ht(
...
+    ht = get_reformatted_info_fields(info_ht)
+    ht = ht.annotate(**{m: ht[m].select(*INFO_FEATURES) for m in info_methods})
+    ht = ht.transmute(**ht.AC_info, **ht.allele_info, **ht.site_info)
+
+    if impute_features:
+        feature_imputed = {}
+        feature_medians = {}
+        for m in info_methods:
+            m_info_ht = ht.select("variant_type", **ht[m])
+            m_info_ht = median_impute_features(
```
Comment (jkgoodrich) [inline gnomad_qc/v4/annotations/generate_variant_qc_annotations.py:494]: oops, I forgot to actually annotate ht with the imputed values...will add tomorrow

**Review by klaricch — CHANGES_REQUESTED** [review]

File: gnomad_qc/v4/annotations/generate_variant_qc_annotations.py [hunk 2]
```diff
@@ -231,42 +241,129 @@ def run_compute_info(
...
+        gvcf_info=mt.gvcf_info.annotate(
+            AS_QUALapprox=recompute_as_qualapprox_from_lpl(mt),
+            **correct_as_annotations(mt),
+        )
+    )
+    ht = correct_mt.select_rows(
         **get_as_info_expr(
-            mt,
+            correct_mt,
+            **AS_INFO_AGG_FIELDS,
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/annotations/generate_variant_qc_annotations.py:268]: it's actually 4 arguments being passed, and this shows that it's all the defaults in gnomad_methods https://github.com/broadinstitute/gnomad_methods/blob/77d2be9a75fd74c0fbffafa7086fb4de5a688203/gnomad/utils/sparse_mt.py#L32

AS_INFO_AGG_FIELDS = {
    "sum_agg_fields": ["AS_QUALapprox", "AS_RAW_MQ"],
    "int32_sum_agg_fields": ["AS_VarDP"],
    "median_agg_fields": ["AS_RAW_ReadPosRankSum", "AS_RAW_MQRankSum"],
    "array_sum_agg_fields": ["AS_SB_TABLE"],
}

If it's OK with you I will keep as is, but I can add a comment if you think that would help

[the remark this answers is not in the document]
Comment (klaricch) [reply gnomad_qc/v4/annotations/generate_variant_qc_annotations.py:268]: ah that makes sense, yeah a comment would help

**Review by klaricch — APPROVED** [review]
lgtm

**Pull request merged**